website/i18n/zh/code.json

```
@@ -30,5 +30,221 @@
   "docs.webpage.title.DocumentSubtitle": {
     "message": "让产品与技术背后的细节更加可视化",
     "description": "We love open source."
+  },
+  "team.webpage.title.Team": {
+    "message": "团队介绍",
+    "description": "Team"
+  },
+  "team.webpage.title.DocumentSubtitle": {
+    "message": "热爱开源的小伙伴们",
```

I would prefer not to translate it, just keep it in English. cc @hf400159 how about you?

```suggestion
    "message": "We love open source.",
```